packages/form-js-editor/src/features/properties-panel/PropertiesPanel.js

```diff
@@ -26,6 +27,7 @@ function getGroups(field, editField) {
 
   const groups = [
     GeneralGroup(field, editField),
+    SerializationGroup(field, editField),
```

Comment:
I think we lack test coverage for this and the other new number property fields, right? Cf. [PropertiesPanel.spec.js](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-editor/test/spec/features/properties-panel/PropertiesPanel.spec.js)

Reply:
Actually it seems we were never testing the number field at all within the properties panel 👀